packages/devtools_app/test/support/utils.dart

```diff
@@ -58,7 +58,7 @@ Future<TreemapNode> loadInstructionSizesJsonAsTree() async {
 
 Future<TreemapNode> loadV8JsonAsTree() async {
   const testDataPath =
-      '../devtools_testing/lib/support/treemap_test_data_v8.json';
+      '../devtools_testing/lib/support/treemap_test_data_v8_new.json';
```

Comment:
won't this affect the other tests? maybe you need to add another helper `loadDiffJsonAsTree`?